charts/catalog/templates/apiregistration.yaml

```diff
@@ -14,5 +14,10 @@ spec:
     namespace: {{ .Release.Namespace }}
     name: {{ template "fullname" . }}-apiserver
   caBundle: {{ .Values.apiserver.tls.ca }}
+  {{ if .Capabilities.APIVersions.Has "apiregistration.k8s.io/v1alpha1" -}}
   priority: 100
+  {{ else if .Capabilities.APIVersions.Has "apiregistration.k8s.io/v1beta1" -}}
+  groupPriorityMinimum: 10000
```

Comment:
@kibbles-n-bytes check out https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1/types.go#L56-L71 for documentation on generally what these should be. I believe that, in a follow-up, we should parameterize these. Not blocking for this PR.